corehq/apps/ota/views.py

```diff
@@ -155,12 +128,61 @@ def search(request, domain):
     return HttpResponse(fixtures, content_type="text/xml; charset=utf-8")
 
 
-def add_blacklisted_owner_ids(search_es, blacklisted_owner_ids):
+def _add_include_closed(search_es, criteria):
+    try:
+        include_closed = criteria.pop('include_closed')
```

Comment:
nit: could use our `string_to_boolean` function and catch a ValueError if you don't want to check `'True'`